weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallsPage/CallsTableButtons.tsx

```diff
@@ -646,6 +647,11 @@ curl '${baseUrl}/calls/stream_query' \\
   return baseCurl;
 }
 
+type PageSizeOption = {
+  readonly value: number;
+  readonly label: string;
+};
+
 export const PaginationButtons = () => {
```

Comment:
We should hide these for the preview tables

<img width="1366" alt="Screenshot 2025-01-17 at 9 27 08 AM" src="https://github.com/user-attachments/assets/d560adfc-1401-480f-b039-f360bb0e8482" />

Reply:
The "child calls" where its just a clickable link doesn't need the pagination buttons

Reply:
Both done